### PR TITLE
[ROCKETMQ-201]Remove spaces before and after the properties

### PR DESCRIPTION
**Pull request by huangyiminghappy**
Remove spaces before and after the properties,Avoid errors due to spaces before and after the properties of data,here is the JIAR link:https://issues.apache.org/jira/browse/ROCKETMQ-201 ,thank you!

### CODE REVIEW

**Comment by huangyiminghappy** [comment]
if i use the file like properties 
image
image

the properties's property has any space,it will get an Not expected value, Numeric possible type conversion error，but The string may get a bug that is not expected,Very difficult to troubleshoot

**Comment by Jaskey** [comment]
Thank you. Please notice your topic and also relate the issue to the jira link https://issues.apache.org/jira/browse/ROCKETMQ.

File: common/src/main/java/org/apache/rocketmq/common/MixAll.java [hunk 1]
```diff
@@ -339,6 +339,8 @@ public static void properties2Object(final Properties p, final Object object) {
                     String key = first.toLowerCase() + tmp;
                     String property = p.getProperty(key);
                     if (property != null) {
+                        // remove spaces before and after
```
Comment (Jaskey) [inline common/src/main/java/org/apache/rocketmq/common/MixAll.java]: I think the purpose is clean enough so I don't think the comment is needed here for `trim()`

Comment (huangyiminghappy) [inline common/src/main/java/org/apache/rocketmq/common/MixAll.java]: ok,i remove the comment

**Comment by Jaskey** [comment]
+1

File: common/src/main/java/org/apache/rocketmq/common/MixAll.java (hunk 1 above)
Comment (Jaskey) [inline common/src/main/java/org/apache/rocketmq/common/MixAll.java]: what about we also trim the method `public static String properties2String(final Properties properties)` ?

**Comment by dongeforever** [comment]
+1

**Comment by dongeforever** [comment]
IMO. if using trim, it is better to trim in all related places

File: common/src/main/java/org/apache/rocketmq/common/MixAll.java [hunk 2]
```diff
@@ -279,7 +279,7 @@ public static String properties2String(final Properties properties) {
         StringBuilder sb = new StringBuilder();
         for (Map.Entry<Object, Object> entry : properties.entrySet()) {
             if (entry.getValue() != null) {
-                sb.append(entry.getKey().toString() + "=" + entry.getValue().toString() + "\n");
+                sb.append(entry.getKey().toString().trim() + "=" + entry.getValue().toString().trim() + "\n");
```
Comment (huangyiminghappy) [inline common/src/main/java/org/apache/rocketmq/common/MixAll.java]: ok，I try it, thanks

Comment (huangyiminghappy) [inline common/src/main/java/org/apache/rocketmq/common/MixAll.java]: @zhouxinyu   i have remove  the above change ,only change the properties's trim(), thank you review it

**Pull request closed without merging**